### PR TITLE
Project 0 

### PR DESCRIPTION
**Pull request by rkm79791**
Seems to work

### CODE REVIEW

File: p0.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    #opposite of to_bohr
+    def to_angstrom(self):
+        if self.units == "bohr" or "Bohr" or "Bohrs" or "bohrs":
+            angstrom_geom = self.geom/1.8897259886
+            print(angstrom_geom)
+
+
+#this just returns the initial string with "new line" stripped
+    def xyz_string(self):
+        return self.info
```
Comment (Bowman205) [inline p0.py]: See if you can recreate the xyz format with this command. Ideally this command would be used to reprint a new .xyz file after changing the molecule.

File: p0.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        
+        masses = list(map(get_mass, atom_labels))
+        self.mass_list = masses
+
+        charges = list(map(get_charge, atom_labels))
+        self.charge_list = charges
+
+    #reads the units of the file and converts from angstrom to bohr if necessary
+    def to_bohr(self):
+        if self.units == "angstrom" or "Angstrom" or "Angstroms" or "angstroms":
```
Comment (Bowman205) [inline p0.py:44]: Good job, way to be robust. I imagine this line could be simplified using regular expressions, but I appreciate the initiative to consider variations in the input file.

File: p0.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    def copy(self): 
+        new_copy = copy_info.deepcopy(info)
+        return new_copy
+
+
+
+if __name__ == "__main__":
+
+#I added some spaces and labels because it made me able to think through the process better and ensure the correct data was printing where desired 
+    my_mol = Molecule("o-benzyne.xyz")
```
Comment (Bowman205) [inline p0.py:75]: It's pretty cool that you used one of the xyz file from your research, but make sure you also push the o-benzyne.xyz as well as p0.xyz. Typically you can add all uncommited files using the git add -A command from the console.

File: p0.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    def to_bohr(self):
+        if self.units == "angstrom" or "Angstrom" or "Angstroms" or "angstroms":
+            bohr_geom = self.geom*1.8897259886
+            print(bohr_geom)
+
+
+    #opposite of to_bohr
+    def to_angstrom(self):
+        if self.units == "bohr" or "Bohr" or "Bohrs" or "bohrs":
+            angstrom_geom = self.geom/1.8897259886
```
Comment (Bowman205) [inline p0.py]: You'll want to change the variable self.units each time you call to_angstrom or to_bohr, that way your code will remember what the current units are.

File: p0.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            print(angstrom_geom)
+
+
+#this just returns the initial string with "new line" stripped
+    def xyz_string(self):
+        return self.info
+
+
+    def copy(self): 
+        new_copy = copy_info.deepcopy(info)
```
Comment (Bowman205) [inline p0.py]: There's several ways to go about this, but I think what you're going for here looks like new_copy = copy.deepcopy(self), but to do this you'll have to add "import copy" with your imports.

File: p0.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        if self.units == "angstrom" or "Angstrom" or "Angstroms" or "angstroms":
+            bohr_geom = self.geom*1.8897259886
+            print(bohr_geom)
+
+
+    #opposite of to_bohr
+    def to_angstrom(self):
+        if self.units == "bohr" or "Bohr" or "Bohrs" or "bohrs":
+            angstrom_geom = self.geom/1.8897259886
+            print(angstrom_geom)
```
Comment (Bowman205) [inline p0.py]: I believe the purpose of the to_angstrom and to_bohr commands is to change the transform the geom between units. So you would want something like self.geom = self.geom/1.889 instead.

**Review by Bowman205 — COMMENTED** [review]
I left some suggestions on how your code could be improved. Pretty good for your first attempt at Python. You should make a directory with the name rkm79791 inside the directory 0, that way we can distinguish your programming project 0 from other programming projects and authors.

File: p0.py (hunk 6 above)
Comment (rkm79791) [reply p0.py]: Thank you for the very helpful feedback on everything! I will get all of the suggestions updated and resubmitted.

File: p0.py (hunk 2 above)
Comment (JonathonMisiewicz) [reply p0.py:44]: I believe you're looking for the .lower() method.

**Pull request closed without merging**